app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsCardsItemFragment.kt

```diff
@@ -129,7 +129,12 @@ class SuggestedEditsCardsItemFragment : SuggestedEditsItemFragment() {
                             val page = response.query?.pages!![0]
                             if (page.imageInfo() != null) {
                                 val imageInfo = page.imageInfo()!!
-                                val title = if (imageInfo.commonsUrl.isEmpty()) page.title else Constants.commonsWikiSite.titleForUri(Uri.parse(imageInfo.commonsUrl)).prefixedText
+                                val title = if (imageInfo.commonsUrl.isEmpty()) {
+                                    page.title
+                                } else {
+                                    val wiki = WikiSite(Service.COMMONS_URL)
```

Comment:
Please use `Constants.commonsWikiSite` instead of creating a new `WikiSite()` object.

Reply:
Thanks for the review! Fixed both of these.